OrbitClientModel/include/OrbitClientModel/CaptureData.h

```diff
@@ -192,10 +192,13 @@ class CaptureData {
 
   [[nodiscard]] const ProcessData* process() const { return &process_; }
 
-  [[nodiscard]] const SamplingProfiler& sampling_profiler() const { return sampling_profiler_; }
+  [[nodiscard]] const PostProcessedSamplingData& post_processed_sampling_data() const {
+    CHECK(post_processed_sampling_data_.has_value());
```

Comment:
I guess OrbitBase/Logging.h should be also included.

Reply:
Done.